site/_filters/filter-by-locale.js

```diff
@@ -0,0 +1,25 @@
+const {defaultLocale} = require('../_data/site.json');
+
+/**
+ * Filters Front Matter data that have the same locale attribute as requested.
+ * @param {FrontMatterData[]} fMData An array of Front Matter data.
+ * @param {string} [locale] Target locale.
+ * @return {FrontMatterData[]}
+ */
+module.exports = function filterByLocale(fMData, locale = defaultLocale) {
+  /** @type Map<string, FrontMatterData> */
+  const filteredFrontMatter = new Map();
+  for (const element of fMData) {
+    if ([locale, defaultLocale].includes(element.locale)) {
+      filteredFrontMatter.set(element.url, element);
```

Comment:
This line seems to add the element no matter what, ignoring the two cases below it.

Reply:
yup, issue with old code that i modified but didnt fully fix.